src/components/todoList/TodoList.scss

```diff
@@ -0,0 +1,143 @@
+* {
+    padding: 0;
+    margin: 0;
+}
+
+body {
+    overflow-y: scroll;
+    overflow-x: hidden;
+    overflow-wrap: break-word;
+}
+
+.myGoals-font-light {
+    padding: 0em 1em 0em 0.8em;
+    margin-left: 0.5em;
+    font-style: normal;
+    font-weight: bolder;
+}
+
+.myGoals-font-dark {
+    color: #cecece;
+    padding: 0em 1em 0em 0.8em;
+    margin-left: 00.5em;
+    font-style: normal;
+    font-weight: bolder;
+}
+
+
+.addTaskDark {
+    background-color: #5f3636;
+    color: #cecece;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    border: none;
+    border-radius: 0.5em;
+    font-weight: bolder !important;
+    font-size: 1.0rem !important;
+    width: 95% !important;
+    margin-left: 0.5em;
+    margin-top: 0.5em !important;
+    text-align: left !important;
+    padding-top: 0.5em !important;
+    padding-bottom: 0.5em !important;
+    padding-left: 0.5em;
+}
+
+.addTaskLight {
+    background-color: #e9b17a;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    border: none;
+    border-radius: 0.5em;
+    font-weight: bolder !important;
+    font-size: 1.0rem !important;
+    width: 95% !important;
+    margin-left: 0.5em;
+    margin-top: 0.5em !important;
+    text-align: left !important;
+    padding-top: 0.5em !important;
+    padding-bottom: 0.5em !important;
+    padding-left: 0.5em;
+}
+
+
+.addTaskDark-time {
```

Comment:
Since we are using scss instead of the restrictive CSS, I would suggest using features like [mixins](https://sass-lang.com/documentation/at-rules/mixin) and scss [variables](https://sass-lang.com/documentation/variables) to write cleaner code with better modularization

Reply:
I'll handle this after making sure that the todo list is functioning as required.